csharp/documentation/library-coverage/coverage.rst

```diff
@@ -0,0 +1,12 @@
+C# framework & library support
+================================
+
+.. csv-table::
+   :header-rows: 1
+   :class: fullWidthTable
+   :widths: auto
+
+   Framework / library,Package,Remote flow sources,Taint & value steps,Sinks (total),`CWE-079` :sub:`Cross-site scripting`
```

Comment:
@yo-h This report explicitly says `Remote flow sources`, and therefore we correctly report 0. But this means that local flow sources (such as `Console.ReadLine()`, which have been migrated to CSV in this PR) are not showing up anywhere in the report. Should we change the generator to report all flow sources instead?

Reply:
I think reporting all flow sources makes sense, to avoid any reporting gaps. For now, reporting all sources in one column seems fine, but in the future we may want to also show a breakdown in the same way as we do for sinks.

Reply:
This change will again introduce some noise on PRs that were opened before this change. The current and previous runs of the job will report different values, and therefore we're going to add a comment to the PRs.